manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/MergeTest.scala

```diff
@@ -228,11 +245,55 @@ return movie""",
         """match (charlie:Person {name:'Charlie Sheen'}), (oliver:Person {name:'Oliver Stone'})
 merge (charlie)-[r:KNOWS]-(oliver)
 return r""",
-      optionalResultExplanation = "Assume that Charlie Sheen and Oliver Stone do not know each other then " +
+      optionalResultExplanation = "As 'Charlie Sheen' and 'Oliver Stone' do not know each other, " +
         "this +MERGE+ query will create a +:KNOWS+ relationship between them. " +
         "The direction of the created relationship is arbitrary.",
       assertions = (p) => assertStats(p, relationshipsCreated = 1)
     )
   }
 
+  @Test def merge_on_a_relationship_between_two_existing_nodes() {
+    testQuery(
+      title = "Merge on a relationship between two existing nodes",
+      text =
+        """+MERGE+ can be used in conjunction with preceding +MATCH+ and +MERGE+ clauses to create a relationship
+between two bound nodes 'm' and 'n', where 'm' is returned by +MATCH+ and 'n' is created or matched by the earlier +MERGE+.""",
+      queryText =
+        """match (person:Person)
+merge (city:City {name: person.bornIn})
+merge (person)-[r:BORN_IN]->(city)
+return person, r, city""".stripMargin,
+      optionalResultExplanation =
+        """This builds on the example from the 'Merge single node derived from existing node property' section above
```

Comment:
I tried this, but no luck. When doing the docbuild, I got an 'invalid content' error...